src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+function getCommentById(id) {
+  return commentsFromServer.filter(comment => comment.postId === id)
+      || null;
+}
```

Comment:
no needs to add null
```suggestion
function getUserById(userId) {
  return usersFromServer.find(user => user.id === userId);
}

function getCommentById(id) {
  return commentsFromServer.filter(comment => comment.postId === id);
}
```